src/bundler.ts

```diff
@@ -278,7 +278,7 @@ class Bundler {
         dom5.remove(htmlImport);
         return;
       }
-      const relative = this._computeRelativeUrl(docUrl, bundleUrl) || bundleUrl;
+      const relative = urlUtils.relativeUrl(docUrl, bundleUrl) || bundleUrl;
```

Comment:
I should mention is that if the relativeUrl can not be calculated because of host/protocol mismatch, it returns bundleUrl.  If bundle.url == targetBundle.url here then relative will be an empty string and will throw.  Can that case occur?

Additionally, if docUrl == bundleUrl then bundleUrl will be returned in full because relative url is an empty string.